test/tests/functional/pbs_cray_vnode_pool.py

```diff
@@ -132,12 +132,12 @@ def test_two_moms_single_vnode_pool(self):
             self.momA.log_match(_msg, max_attempts=9, starttime=start_time)
             found_in_momA = 1
         except PtlLogMatchError:
-            pass
+            found_in_momA = 0
         try:
             self.momB.log_match(_msg, max_attempts=9, starttime=start_time)
             found_in_momB = 1
         except PtlLogMatchError:
-            pass
+            found_in_momB = 0
```

Comment:
While it doesn't hurt, there is no need to initialize found_in_mom[AB] any more.

Reply:
Yup. It makes more sense. I will update accordingly.